google_analytics/utils.py

```diff
@@ -127,6 +128,13 @@ def build_ga_params(
     # construct the gif hit url
     ga_url = "http://www.google-analytics.com/collect"
     utm_url = ga_url + "?&" + urlencode(params)
+    ga_logging_enabled = False
+    if hasattr(settings, 'ENABLE_GA_LOGGING'):
```

Comment:
I still think you should just use `if settings.get('...', False):` here. Because it removes the need for the if statement below

Reply:
When I do that the tests give me the following error: ` val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'get'` 

@miltontony do you know why?

Reply:
Yeah, you always have to use the `hasattr`. Django settings is not a dict in that sense. But I usually have `if hasattr(settings, 'FIELD') and settings.FIELD` in 1 line, rather than a nested if.